.eslintrc.json

```diff
@@ -0,0 +1,43 @@
+{
+    "env": {
+        "es6": true,
+        "node": true,
+        "jasmine": true
+    },
+    "extends": [
+        "standard"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "no-unused-vars": "off",
+        "@typescript-eslint/no-unused-vars": "error",
+        "quotes": [
+            "error",
+            "single",
+            {
+                "avoidEscape": true
+            }
+        ],
+        "max-lines": [
+            "error",
+            160
+        ],
+        "func-style": [
+            "error",
+            "expression"
+        ],
+        "semi": "off",
```

Comment:
Should be turned on?

Reply:
You are right. I forgot to change here.
Now I turned on the rule.